bot/exts/evergreen/realpython.py

```diff
@@ -56,7 +56,7 @@ async def realpython(self, ctx: commands.Context, *, user_search: str) -> None:
         article_embed = Embed(
             title="Search results - Real Python",
             url=SEARCH_URL.format(user_search=quote_plus(user_search)),
-            description="Here are the top 5 results:",
+            description=f"Here are the top {len(articles)} results:",
             color=Colours.orange,
         )
```

Comment:
```suggestion
        if len(articles) == 1:
            description = "Here is the result:"
        else:
            description = f"Here are the top {len(articles)} results:"

        article_embed = Embed(
            title="Search results - Real Python",
            url=SEARCH_URL.format(user_search=quote_plus(user_search)),
            description=description,
            color=Colours.orange,
        )
```

Reply:
GitHub doesn't want to cooperate 👀

Reply:
LGTM

Reply:
![image](https://user-images.githubusercontent.com/38372706/131387702-f53b81c9-eb67-40e8-b5ee-b87286b84967.png)

It is better if @brad90four simply makes the commit locally.

Reply:
@Bluenix2 Yeah, this is what I was talking about GitHub and cooperation :eyes: